kafka-rest/src/main/java/io/confluent/kafkarest/controllers/RecordSerializerFacade.java

```diff
@@ -28,14 +28,14 @@
 final class RecordSerializerFacade implements RecordSerializer {
 
   private final NoSchemaRecordSerializer noSchemaRecordSerializer;
-  private final Provider<SchemaRecordSerializer> schemaRecordSerializer;
+  private final Provider<SchemaRecordSerializer> schemaRecordSerializerProvider;
```

Comment:
You can completely revert this file, right? There's only the attribute rename now, which is not needed anymore.

Reply:
True but it is a provider, so it doesn't really matter.  leaving.